tests/allowed_addresses.go

```diff
@@ -172,12 +172,19 @@ func AllowedAddresses(network interfaces.LocalNetwork) {
 
 	// Test Relayer 1
 	log.Info("Testing Relayer 1: All sources -> All destinations")
-	relayerCleanup := testUtils.BuildAndRunRelayerExecutable(ctx, relayerConfigPath1)
+	relayerCleanup, readyChan := testUtils.RunRelayerExecutable(
+		ctx,
+		relayerConfigPath1,
+		fmt.Sprintf(
+			"http://localhost:%d/health",
+			relayerConfig1.APIPort,
```

Comment:
Not a blocker but since we can assume that all of these are running on localhost I have a slight preference to just pass in the port and handle the URL construction on the other side. That would be assuming that we have the same path for all healthcheck endpoints but I don't think that's too constraining.

Reply:
A more extensible option would be to pass the config object in its entirety, and grab the relevant fields inside the function.